paper/paper.md

```diff
@@ -1,42 +1,45 @@
 ---
-title: 'gamut: A Geospatial Analysis of Multisector Urban Teleconnections'
+title: 'gamut: A Geospatial R Package to Analyze Multisector Urban Teleconnections'
 tags:
-  - R
-  - Multisector Dynamics
-  - Water
-  - Energy
-  - Land
-  - Urban
-  - Geospatial
+- R
+- Multisector Dynamics
+- Water
+- Energy
+- Land
+- Urban
+- Geospatial
+date: "30 September 2021"
+output:
+  html_document: 
+    df_print: paged
 authors:
-  - name: Kristian D. Nelson
-    orcid: 0000-0002-6745-167X
-    affiliation: 1
-  - name: Sean W. Turner
-    orcid: 0000-0003-4400-9800
-    affiliation: 1
-  - name: Chris Vernon
-    orcid: 0000-0002-3406-6214
-    affiliation: 1
-  - name: Jennie Rice
-    orcid: 0000-0002-7833-9456
-    affiliation: 1
-affiliations:
- - name: Pacific Northwest National Laboratory
-   index: 1
-date: 26 April 2020
+- name: Kristian D. Nelson
+  orcid: 0000-0002-6745-167X
+  affiliation: 1
+- name: Sean W. Turner
+  orcid: 0000-0003-4400-9800
+  affiliation: 1
+- name: Chris Vernon
+  orcid: 0000-0002-3406-6214
+  affiliation: 1
+- name: Jennie Rice
+  orcid: 0000-0002-7833-9456
+  affiliation: 1
 bibliography: paper.bib
+affiliations:
+- name: Pacific Northwest National Laboratory
```

Comment:
Affiliation should be `Pacific Northwest National Laboratory, Richland, Washington, USA`